There's an unrelated build failure:

~~~
/var/lib/buildkite-agent/zephyr-module-cache/modules/lib/mcumgr/cmd/img_mgmt/port/zephyr/src/zephyr_img_mgmt.c:334:26: error: initialization discards 'const' qualifier from pointer target type [-Werror=discarded-qualifiers]
334 |     struct device *dev = flash_area_get_device(fa);
    |                          ^~~~~~~~~~~~~~~~~~~~~
~~~

@tbursztyka: FYI, I believe it's related to your recent "device constify" patch, so checking if it's a known issue in works.

@legoabram: Please be ready to rebase this on later master to get the CI pass.

--- a/drivers/wifi/winc1500/wifi_winc1500.c
+++ b/drivers/wifi/winc1500/wifi_winc1500.c
@@ -298,19 +298,21 @@ static int winc1500_get(sa_family_t family,
 			struct net_context **context)
 {
 	struct socket_data *sd;
+	SOCKET sock;
 
 	if (family != AF_INET) {
 		LOG_ERR("Only AF_INET is supported!");
 		return -1;
 	}
 
-	(*context)->offload_context = (void *)(sint32)socket(family, type, 0);
-	if ((*context)->offload_context < 0) {
+	sock = socket(family, type, 0);
+	if (sock < 0) {
 		LOG_ERR("socket error!");
 		return -1;
 	}
 
-	sd = &w1500_data.socket_data[(int)(*context)->offload_context];
+	(*context)->offload_context = (void *)(intptr_t)sock;
+	sd = &w1500_data.socket_data[sock];
 
 	k_sem_init(&sd->wait_sem, 0, 1);
 

